src/App.js

```diff
@@ -1055,11 +1058,11 @@ export class App extends React.Component<AppProps, AppState> {
         return commandBlocks;
     }
 
-    makeStartingCharacterState(world: WorldName, startingX: number, startingY: number): CharacterState {
+    makeStartingCharacterState(world: WorldName, startingDirection: number, startingX: number, startingY: number): CharacterState {
```

Comment:
We can remove the `world` parameter from `makeStartingCharacterState` as we are no longer using it. Previously we used it to retrieve the direction, but we are now passing the direction in explicitly.